### DISCUSSION TITLE
Put Letter on QR Code C#

### DISCUSSION THREAD

**Discussion by Ratanaly** [micjahn/ZXing.Net#584] [Q&A]
How to put Letter of Link on QR code ? 

Thank You

**Comment by micjahn** [comment]
What do you mean exactly? Do you have an example that shows what you want to achieve?

**Comment by Ratanaly** [comment]
attachment below Sir …

**Reply by micjahn** [reply]
I don't find any attachement...

**Reply by Ratanaly** [reply]
![image](https://github.com/user-attachments/assets/1c7cd061-460b-432e-8dbc-f9a0677082ca)

**Comment by Ratanaly** [comment]
I will resend it.

**Reply by Ratanaly** [reply]
![image](https://github.com/user-attachments/assets/413971a9-6699-4eb1-9991-e97a40e8baf5)

**Reply by micjahn** [reply]
ZXing.Net doesn't support it. You have to add it manually. Generate the QR code with the highest error correction level and add the text with your favourite image manipulation library.
